.template/variants/web/app/javascript/build.js

```diff
@@ -2,7 +2,10 @@ const watch = process.argv.slice(2).includes('--watch');
 
 require('esbuild')
   .context({
-    entryPoints: ['app/javascript/application.js'],
+    entryPoints: [
+      { in: 'app/javascript/application.js', out: 'application.js' },
```

Comment:
also take this changes to add this placeholder for engine JS file, so the future developer no need to spend time to figure out this.

I have tested it in our project

<img width="1382" alt="image" src="https://github.com/nimblehq/rails-templates/assets/11751745/52e3a04f-f7b4-4266-b31c-48e550c8b975">